src/server/appHandler.js

```diff
@@ -82,8 +82,10 @@ function renderApp (res, renderProps, history, store) {
     )
 
     let state = store.getState()
+    let unreadCount = get(state, 'people.current.new_notification_count')
 
     return React.createElement(Html, {
+      pageTitle: 'Hylo' + (unreadCount && ` (${unreadCount})`),
```

Comment:
http://stackoverflow.com/questions/30280816/whose-job-is-it-in-flux-react-to-set-the-page-title

This approach feels ok to me -- set the title in the store, add the value
to the connect decorator in the App component, then create a
componentWillReceiveProps method that just changes the page title with
vanilla JavaScript when the prop changes.
On Wed, Feb 24, 2016 at 4:40 PM Robbie Carlton <notifications@github.com>
wrote:

> In src/server/appHandler.js
> <https://github.com/Hylozoic/hylo-redux/pull/26#discussion_r54032604>:
>
> >
> >      return React.createElement(Html, {
> > +      pageTitle: 'Hylo' + (unreadCount && ` (${unreadCount})`),
>
> this is not updating when the store updates, because it's outside Redux.
> Any thoughts on how to get this updating? One possibility would be doing it
> with JS in eg the App component
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Hylozoic/hylo-redux/pull/26/files#r54032604>.
>